src/Compilers/Core/Portable/InternalUtilities/PEReaderExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.CodeAnalysis.InternalUtilities
+{
+    internal static class PEReaderExtensions
```

Comment:
>PEReaderExtensions [](start = 26, length = 18)

I'd prefer not having these extension methods. They are only used in two call sites, one of those is in tests. That don't really save that much and their use is very specific.

Reply:
I added them because they are used in test code and the code that it is testing, so this helps make sure they always behave the same. 

That said, I'm not too picky about it so if it's a problem I'll remove and duplicate in both places.

Reply:
The purpose of test code is to validate expectations and prevent regressions. Sharing code between test and product is detrimental to that purpose. E.g. if these extension methods were changed in future by mistake and a bug is introduced the tests would pass.